grafanalib/_gen.py

```diff
@@ -124,3 +128,34 @@ def generate_dashboards_script():
 def generate_dashboard_script():
     """Entry point for generate-dasboard."""
     run_script(generate_dashboard)
+
+
+def parse_dashboard(args):
+    parser = argparse.ArgumentParser(prog='generate-dashboard')
+    # parser.add_argument(
+    #     '--output', '-o', type=os.path.abspath,
+    #     help='Where to write the dashboard JSON'
+    # )
```

Comment:
Please delete commented out code before merging.

Reply:
This part is actually not done yet ;-)